src/main/java/game/GameEngine.java

```diff
@@ -1,6 +1,7 @@
 package game;
 
 import game.map.Map;
+import game.map.MapShower;
```

Comment:
I don't see `MapShower` class anywhere. Did you forget to commit it?

Reply:
added